api/users/repositories/organizationReopository.go

```diff
@@ -0,0 +1,43 @@
+package repositories
+
+import (
+	"context"
+
+	"github.com/null-channel/eddington/api/infrastrucure"
+	models "github.com/null-channel/eddington/api/users/models"
+)
+
+type OrgReposiotry struct {
+	infrastrucure.Database
+}
+
+func (repository *OrgReposiotry) Seed() error {
+	_, err := repository.DB().NewCreateTable().Model((*models.Org)(nil)).Exec(context.Background())
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func (repository *OrgReposiotry) GetOrgByOwnerId(ctx context.Context, userId string) ([]*models.Org, error) {
+	var orgs []*models.Org
+	err := repository.DB().NewSelect().
+		Model(&orgs).
+		Where("owner_id = ?", userId).
+		Scan(ctx, &orgs)
+
+	if err != nil {
+		return nil, err
+	}
+
+	return orgs, nil
+}
+
+func (repository *OrgReposiotry) Save(ctx context.Context, org *models.Org) error {
+	_, err := repository.DB().NewInsert().
+		Model(org).
+		On("CONFLICT (owner_id) DO UPDATE").
```

Comment:
not 100% sure I know why my branch works but..... `owner_id` is not unique so it could match many records and this operation. a user in therory (as the database is designed can own two orgs.) This is probably currently a bug. You will need to probably use the pk id.  That will probably mean you need to retrive the orgs for a user and then update or create this one.